.github/scripts/close_nonexistent_disable_issues.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import json
+import multiprocessing as mp
+import os
+import re
+import tempfile
+from typing import Any, Dict, List, Optional, Tuple
+
+import requests
+import rockset  # type: ignore[import]
+
+LOGS_QUERY = """
+with
+    shas as (
+        SELECT
+            push.head_commit.id as sha,
+        FROM
+            commons.push
+        WHERE
+            push.ref = 'refs/heads/viable/strict'
+            AND push.repository.full_name = 'pytorch/pytorch'
+        ORDER BY
+            push._event_time DESC
+        LIMIT
+            5
+    )
+select
+    id,
+    name
+from
+    workflow_job j
+    join shas on shas.sha = j.head_sha
+where
+    j.name like '% / test%'
+    and j.name not like '%rerun_disabled_tests%'
+    and j.name not like '%mem_leak_check%'
+"""
+
+TEST_EXISTS_QUERY = """
+select
+    count(*) as c
+from
+    test_run_s3
+where
+    cast(name as string) like :name
+    and classname like :classname
+    and _event_time > CURRENT_TIMESTAMP() - DAYS(7)
+"""
+CLOSING_COMMENT = (
+    "I cannot find any mention of this test in rockset for the past 7 days "
+    "or in the logs for the past 5 commits on viable/strict.  Closing this "
+    "issue as it is highly likely that this test has either bee renamed or removed."
```

Comment:
Nit: Mini typo.  Also we could tell people to reopen the issue if they think it's a false positive